website/src/layouts/internals/layout-toc.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+import { Spacings } from '@commercetools-frontend/ui-kit';
+import * as colors from '../../colors';
+
+const itemType = {
+  url: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+};
+const itemsType = PropTypes.arrayOf(
+  PropTypes.shape({
+    ...itemType,
+    items: PropTypes.arrayOf(PropTypes.shape(itemType)),
+  })
+);
+
+const Link = styled.a`
+  font-size: ${props => (props.size === 's' ? '0.5rem' : '0.8rem')};
+  line-height: 1.1;
+  color: ${colors.light.text};
+  text-decoration: none;
+
+  :hover {
+    color: ${colors.light.primarySoft};
+    text-decoration: underline;
+  }
+  :hover,
+  :active {
+    outline-width: 0;
```

Comment:
😢

Reply:
?

Reply:
Why are we removing the outline? makes it really hard to use keyboard to navigate

Reply:
The focus outline for a11y is still there, this is just for hover/active state.

![2019-05-02 17 16 39](https://user-images.githubusercontent.com/1110551/57125895-5f675500-6d8b-11e9-93df-a70824dd165c.gif)